app/models/agents/website_agent.rb

```diff
@@ -166,6 +169,25 @@ def validate_options
       validate_web_request_options!
     end
 
+    def validate_consider_http_success_option!
+      consider_success = options["consider_http_error_success"]
+      if consider_success != nil
```

Comment:
The reason I have the code as `!= nil` was so that empty arrays would not be considered valid. Related to strings, is this something that you have in mind: `[404, "422"]` ?